src/Extensions/Sitemap/IOSitemapPattern.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace IO\Extensions\SiteMap;
+
+use IO\Services\TemplateConfigService;
+use Plenty\Modules\Plugin\Contracts\PluginRepositoryContract;
+use Plenty\Modules\Plugin\Events\LoadSitemapPattern;
+use Plenty\Modules\Plugin\Models\Plugin;
+use Plenty\Modules\Plugin\Services\PluginSeoSitemapService;
+use Plenty\Plugin\ConfigRepository;
+
+class IOSitemapPattern
+{
+    private $contentRoutes = [
+        'cancellation-rights',
+        'cancellation-form',
+        'legal-disclosure',
+        'privacy-policy',
+        'gtc',
+        'contact'
+    ];
+    
+    /**
+     * @param LoadSitemapPattern $sitemapPattern
+     */
+    public function handle(LoadSitemapPattern $sitemapPattern)
+    {
+        /** @var PluginRepositoryContract $pluginRepository */
+        $pluginRepository = pluginApp(PluginRepositoryContract::class);
+        
+        $pluginList = $pluginRepository->searchPlugins(['name' => 'IO'])->getResult();
+        $plugin = $pluginList[0];
+        
+        if($plugin instanceof Plugin && $plugin->activeProductive)
```

Comment:
Hier muss unbedingt mit Core geklärt werden, wie die Abfrage aussehen muss, sobald die Plugin-Sets veröffentlicht sind. Stage und Productive wird es in Zukunft ja nicht mehr geben!
Bitte eine entsprechende Karte ins Backlog packen, damit das nicht vergessen wird!

Reply:
Habe eine Karte zum Backlog hinzugefügt.